pupil-spa/src/app/familiarisation-area/familiarisation-area.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { StorageService } from '../services/storage/storage.service';
+import { QuestionService } from '../services/question/question.service';
+import { Pupil } from '../pupil';
+import { AccessArrangements, accessArrangementsDataKey } from '../access-arrangements';
+import { Config } from '../config.model';
+
+@Component({
+  selector: 'app-familiarisation-area',
+  templateUrl: './familiarisation-area.component.html',
+  styleUrls: ['./familiarisation-area.component.scss']
+})
+export class FamiliarisationAreaComponent {
+  private pupil: Pupil;
+  private selectedSize;
+
+  private fontSettings = [
```

Comment:
@danwilliams shouldn't the fontSettings be part of a config entity along with the other access arrangement config options?

Reply:
That sounds better - I’ve moved them into an AccessArrangementsConfig 👍